### PR TITLE
Created a Remove-PowerBIReport cmdlet

### PR DESCRIPTION
**Pull request by carmbell**
Created a Remove-PowerBIReport cmdlet and tests

### CODE REVIEW

File: src/Modules/Reports/Commands.Reports.Test/RemovePowerBIReportTests.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+using System.Management.Automation;
+using Microsoft.PowerBI.Commands.Common.Test;
+using Microsoft.PowerBI.Commands.Profile.Test;
+using Microsoft.PowerBI.Commands.Reports;
+using Microsoft.PowerBI.Common.Abstractions;
+using Microsoft.PowerBI.Common.Api;
+using Microsoft.PowerBI.Common.Api.Datasets;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
```
Comment (CodeCyclone) [inline src/Modules/Reports/Commands.Reports.Test/RemovePowerBIReportTests.cs]: Delete line

File: src/Modules/Reports/Commands.Reports/RemovePowerBIReport.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+using Microsoft.PowerBI.Common.Api.Reports;
+using Microsoft.PowerBI.Common.Api.Workspaces;
+using Microsoft.PowerBI.Common.Api.Datasets;
+using Microsoft.PowerBI.Common.Client;
+
+namespace Microsoft.PowerBI.Commands.Reports
+{
+    [Cmdlet(CmdletVerb, CmdletName)]
+    [OutputType(typeof(void))]
+
```
Comment (CodeCyclone) [inline src/Modules/Reports/Commands.Reports/RemovePowerBIReport.cs]: Delete line

File: src/Modules/Reports/Commands.Reports/RemovePowerBIReport.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        public Guid Id { get; set; }
+
+        [Parameter(Mandatory = false)]
+        [Alias("GroupId")]
+        public Guid WorkspaceId { get; set; }
+
+        public RemovePowerBIReport() : base() { }
+
+        public RemovePowerBIReport(IPowerBIClientCmdletInitFactory init) : base(init) { }
+
```
Comment (CodeCyclone) [inline src/Modules/Reports/Commands.Reports/RemovePowerBIReport.cs]: Delete lines

File: src/Modules/Reports/Commands.Reports/RemovePowerBIReport.cs [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+        protected override void BeginProcessing()
+        {
+            base.BeginProcessing();
+        }
+
+
```
Comment (CodeCyclone) [inline src/Modules/Reports/Commands.Reports/RemovePowerBIReport.cs]: delete line

File: src/Modules/Reports/Commands.Reports/help/MicrosoftPowerBIMgmt.Reports.md [hunk 5]
```diff
@@ -38,3 +38,6 @@ Creates a new empty Power BI dashboard.
 ### [New-PowerBIReport](New-PowerBIReport.md)
 Creates a Power BI report.
 
+### [Remove-PowerBIReport](Remove-PowerBIReport.md)
+{{Fill in the Synopsis}}
```
Comment (CodeCyclone) [inline src/Modules/Reports/Commands.Reports/help/MicrosoftPowerBIMgmt.Reports.md]: Fill out

File: src/Modules/Reports/Commands.Reports/help/Remove-PowerBIReport.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+external help file: Microsoft.PowerBI.Commands.Reports.dll-Help.xml
+Module Name: MicrosoftPowerBIMgmt.Reports
+online version: https://docs.microsoft.com/en-us/powershell/module/microsoftpowerbimgmt.reports/remove-powerbireport?view=powerbi-ps
+schema: 2.0.0
+---
+
+# Remove-PowerBIReport
+
+## SYNOPSIS
+{{Fill in the Synopsis}}
```
Comment (CodeCyclone) [inline src/Modules/Reports/Commands.Reports/help/Remove-PowerBIReport.md]: Fill out file

File: src/Modules/Reports/Commands.Reports/help/MicrosoftPowerBIMgmt.Reports.md (hunk 5 above)
Comment (CodeCyclone) [reply src/Modules/Reports/Commands.Reports/help/MicrosoftPowerBIMgmt.Reports.md]: Ignore this one, gets generated

**Comment by carmbell** [comment]
Changes committed

**Pull request merged**